torch/_dynamo/trace_rules.py

```diff
@@ -2200,12 +2209,8 @@
         "torch._ops.dl_open_guard",
         "torch._ops.get_cached_ops",
         "torch._ops.key_extractor",
-        "torch._ops.mode_stack_per_key",
-        "torch._ops.pop_mode_for_key",
-        "torch._ops.push_mode_for_key",
         "torch._ops.reset_cached_ops",
         "torch._ops.resolve_key",
-        "torch._ops.temporarily_pop_mode",
```

Comment:
These functions has been removed from PyTorch source code, we should remove them from this list when the removing PR got merged. But due to CI skipped ```test_torch_name_rule_map_updated``` unit test recently, we didn't catch this. I'll follow up with dev infra to enable it.